.eslintrc.json

```diff
@@ -12,6 +12,11 @@
         "no-multiple-empty-lines": ["error", {"max": 2, "maxEOF": 0}],
         "spaced-comment": ["error", "always", {"markers": ["/"]}]
       }
+    },
+    {
+      "files": ["*.mjs"],
+      "env": {"es6": true},
+      "parser": "babel-eslint"
```

Comment:
We need a special parser for our files, because `import()` isn't an officially supported part of the language yet. The work around here is that we "pretend" to have written our source for Babel, while in reality we've written in for `node --experimental-modules`.